packages/react/package.json

```diff
@@ -33,7 +33,10 @@
   "sideEffects": false,
   "scripts": {
     "typecheck": "tsc --noEmit",
-    "build": "rm -rf dist && mkdir -p dist/cjs dist/esm auto/polaris auto/mui && echo '{\"type\": \"commonjs\"}' > dist/cjs/package.json && echo '{\"main\": \"../../dist/cjs/auto/polaris/index.js\"}' > auto/polaris/package.json && echo '{\"main\": \"../../dist/cjs/auto/mui/index.js\"}' > auto/mui/package.json && echo '{\"type\": \"module\"}' > dist/esm/package.json && tsc -b tsconfig.cjs.json tsconfig.esm.json ",
+    "clean": "rimraf /dist /auto",
+    "prebuild": "mkdir -p dist/cjs dist/esm auto/polaris auto/mui && echo '{\"type\": \"commonjs\"}' > dist/cjs/package.json && echo '{\"main\": \"../../dist/cjs/auto/polaris/index.js\"}' > auto/polaris/package.json && echo '{\"main\": \"../../dist/cjs/auto/mui/index.js\"}' > auto/mui/package.json && echo '{\"type\": \"module\"}' > dist/esm/package.json",
+    "copy": "copyfiles -u 1 src/**/*.css dist/esm && copyfiles -u 1 src/**/*.css dist/cjs",
```

Comment:
Perhaps this should be called something like `copyCssFiles` so that people don't get the impression that this copies more than it actually does